tests/phpunit/CRM/Activity/Form/SearchTest.php

```diff
@@ -49,7 +49,6 @@ public function testSearch() {
         'source_record_id' => '1',
         'activity_type_id' => '6',
         'activity_type' => 'Contribution',
-        'activity_is_test' => '0',
```

Comment:
@colemanw this suggests that passing

```
      'is_test' => $activity->is_test,
```

is passing null or something & resulting in the db default not being applied? I can't quite think my way through what's happening here